tests/puppeteer/campaigns/sanity/catalogFO/01_filterProducts.js

```diff
@@ -28,19 +28,19 @@ global.scenario('Filter Products by categories in Home page', () => {
 
   test('should check and get the products number', async () => {
     await homePage.waitForSelectorAndClick(homePage.allProductLink);
-    allProductsNumber = await homePage.getNumberOfProducts();
+    allProductsNumber = await homePage.getNumberFromText(homePage.totalProducts);
```

Comment:
Maybe add comment here to say that `getNumberFromText` is used to get Number Of Products